packages/scoutgame/src/wallets/updatePrimaryWallet.ts

```diff
@@ -0,0 +1,25 @@
+import { prisma } from '@charmverse/core/prisma-client';
+import { getAddress } from 'viem/utils';
+
+export async function updatePrimaryWallet(_address: string, userId: string) {
+  const address = getAddress(_address).toLowerCase();
+
+  const userWallet = await prisma.scoutWallet.findUnique({
```

Comment:
better to use a count than findUnique, since findUnique retursn data we dont need, and can break if we do a db migration and the schema doesnt line up

Reply:
Agree.